kratos/geometries/line_3d_2.h

```
@@ -671,12 +671,6 @@ class Line3D2 : public Geometry<TPointType>
     {
         // We define the normal
         array_1d<double,3> normal = ZeroVector(3);
-        
-        // We get the local points
-        const TPointType& first_point  = BaseType::GetPoint(0);
-        const TPointType& second_point = BaseType::GetPoint(1);
- 
-        // TODO: FINISH ME
  
         // We normalize
         const double norm_normal = norm_2(normal);
```

you should throw an error if the normal's norm is zero. AT LEAST IN DEBUG

it means that the geometry is degradated